src/components/bulk-email-tool/bulk-email-task-manager/BulkEmailContentHistory.jsx

```diff
@@ -1,8 +1,23 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
 
 import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { getSentEmailHistory } from './api';
```

Comment:
What is `./api`? The name seems very generic, is it the API of a library provided by `frontend-app-communications`?

Reply:
That is referencing a local `api.js` file for the `bulk-email-task-manager` component itself. The pattern we follow, as I understand it, is that each component can have their own `api.js` that contains whatever REST API functions needed that the component will rely on.